apps/client/src/components/User/UserProfileCardControls.svelte

```diff
@@ -1,7 +1,21 @@
-<script>
-    export let user;
+<script lang="ts">
+  export let user;
+  export let currentUser: Boolean;
 </script>
+
 <div class="card-actions justify-between">
+  {#if currentUser}
+    <div class="gap-2">
+      <a href="/settings" class="btn btn-sm btn-outline gap-2">
+        <iconify-icon icon="ri:settings-3-line" />
+        Settings
+      </a>
+      <a href="/authored" class="btn btn-sm btn-primary gap-2">
+        <iconify-icon icon="ri:book-read-line" />
```

Comment:
What if the user doesn't exist?

Reply:
I've just encountered a situation where a user doesn't exist in the DB but is authenticated via GitHub. the custom Adapter should help us resolve this problem by automatically creating a `user` row upon signup.